---
apiVersion: "v1"
kind: "Event"
metadata:
  labels:
    EntandoResourceKind: "TestResource"
    EntandoResourceNamespace: "my-namespace"
    TestResource: "my-app"
  name: "my-app-59138143"
  namespace: "my-namespace"
  ownerReferences:
  - apiVersion: "test.org/v1"
    kind: "TestResource"
    blockOwnerDeletion: true
    controller: true
    name: "my-app"
    uid: "fdcbf36b-3bb8-46a3-9a41-0d2c577f58cb"
action: "FAILED"
count: 1
firstTimestamp: "2021-07-01T07:08:34Z"
involvedObject:
  apiVersion: "test.org/v1"
  kind: "TestResource"
  fieldPath: "status"
  name: "my-app"
  namespace: "my-namespace"
  resourceVersion: "9784691"
  uid: "fdcbf36b-3bb8-46a3-9a41-0d2c577f58cb"
lastTimestamp: "2021-07-01T07:08:34Z"
message: "The deployment of TestResource my-namespace/my-app failed due to Deployment\
  \ failed. Please inspect the logs of the pod my-namespace/my-app-deployment. Fix\
  \ the root cause and then trigger a redeployment by adding the annotation 'entando.org/processing-instruction:\
  \ force'"
reason: "Failed"
related:
  apiVersion: "v1"
  kind: "Pod"
  name: "test-controller-pod"
  namespace: "controller-namespace"
source:
  component: "entando-k8s-ource-controller"
type: "Error"
